HISTORY.rst

```diff
@@ -2,6 +2,63 @@
 History
 =======
 
+2.9.0 (2022-07-13)
+------------------
+
+* Dropped support for Python 3.6 and 3.7.
```

Comment:
```suggestion
* Dropped support for Python 3.5, 3.6 and 3.7.
```

Reply:
I see that 3.5 support was "dropped" already in version 2.8, according to the changelog, however I confirm that Python 3.5 was still documented as supported according to Python classifiers and `python_requires`. See 
https://github.com/yt-project/unyt/blob/eeefa00a2fddbf0dba6ab854e968ef43e31f851e/setup.py#L51

Reply:
To reduce confusion, I'm going to leave this line as-is.

Reply:
Fine by me